src/main/msp/msp.c

```diff
@@ -4344,6 +4358,8 @@ static mspResult_e mspCommonProcessInCommand(mspDescriptor_t srcDesc, int16_t cm
 #endif //USE_OSD_HD
 #endif // OSD
 
+    case
```

Comment:
Missing statement - probably leftover?

Reply:
I have no idea how that got there, most probably VSCode tricked me again. Looks like it is now fixed